.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - main
+
+env:
+ CI: false
+ REACT_APP_SERVICE_ID: ${{ secrets.REACT_APP_SERVICE_ID }}
+ REACT_APP_TEMPLATE_ID: ${{ secrets.REACT_APP_TEMPLATE_ID }}
+ REACT_APP_EMAILJS_PUBLIC_KEY: ${{ secrets.REACT_APP_EMAILJS_PUBLIC_KEY }}
+ REACT_APP_SITE_KEY: ${{ secrets.REACT_APP_SITE_KEY }}
+ REACT_APP_SECTET_KEY: ${{ secrets.REACT_APP_SECTET_KEY }}
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Install and Build 🏗️
+        run: |
+          npm ci
+          npm run build
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.7
+        with:
+          branch: gh-pages
```

Comment:
do we need it? :)

Reply:
Yes, we do  :) this branch is used by netlify, thanks to that netlify don't need to build app and just serve static files from this branch